src/data/javascript-quiz.ts

```diff
@@ -1710,6 +1710,30 @@ const javascriptQuiz = [
     Link:
       "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN",
   },
+  {
+    Question:
+      "In JavaScript, what is the result for typeof NaN?",
+    Answer: "number",
+    Distractor1: "Object",
+    Distractor2: "String",
+    Distractor3: "Undefined ",
+    Explanation:
+      "typeof NaN will return number because NaN is a property of the Number object in JavaScript",
+    Link:
+      "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN",
```

Comment:
This question is a duplicate.
Please remove it

Reply:
@sinhashubhashish 

You don't need to open up a new PR.
You can push up new changes to remove this question here and it will be updated in the PR